bedmaker.py

```diff
@@ -142,6 +149,44 @@ def main():
             cmd = [cmd]
         cmd.append(gzip_cmd)
     pm.run(cmd, target=args.output_file)
+
+    print("Generateing bigBED files for {}".format(args.input_file))
+    bedfile_name = os.path.split(args.output_file)[1]
+    fileid = os.path.splitext(os.path.splitext(bedfile_name)[0])[0]
+    # Produce bigBed (bigNarrowPeak) file from peak file 
+    bigNarrowPeak = os.path.join(args.output_bigbed, fileid + ".bigBed")
+    if args.input_type != "bigBed":
+        temp = tempfile.NamedTemporaryFile(dir=args.output_bigbed, delete=False)
+        if not os.path.exists(bigNarrowPeak):
+            df = pd.read_csv(args.output_file, sep='\t', header=None,
```

Comment:
AFAIK this is the only spot we use pandas library across the bedbase project. Why don't we read the CSV using standard Python libs so we don't require the installation of all the pandas-related dependencies?

see this issue: https://github.com/databio/bedhost/issues/35

Reply:
I think that's a good idea...but taken further, why are we reading the file into memory in the first place? Is this really necessary? If we're just converting then we don''t need to load the data into memory in python, do we?

Reply:
I feel like you might be able to do this more efficiently with awk/sed or something.